fmriprep/workflows/bold/base.py

```diff
@@ -0,0 +1,785 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Orchestrating the BOLD-preprocessing workflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autofunction:: init_func_preproc_wf
+.. autofunction:: init_func_reports_wf
+.. autofunction:: init_func_derivatives_wf
```

Comment:
Do you really want to include reports and derivatives in the API? I think I skipped them. But I suppose no harm, either.

Reply:
It generates two entries with one-liner description. May even help to newcomers.